packages/libpq/build

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# libpq is the postgresql client library. It is required to communicate with any
+# database that communicates using the postgresql dialect, for example CockroachDB.
+
```

Comment:
Again, do we need `set -exu` ?

Reply:
okay, I added this, and removed `find /opt/mesosphere`. CC @gpaul